src/workerd/io/compatibility-date.capnp

```diff
@@ -424,12 +424,20 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   pythonWorkers @43 :Bool
       $compatEnableFlag("python_workers")
       $pythonSnapshotRelease(pyodide = "0.26.0a2", pyodideRevision = "2024-03-01",
-          packages = "2024-03-01", backport = 0);
+          packages = "2024-03-01", backport = 0)
+      $impliedByAfterDate(name = "pythonWorkersDevPyodide", date = "2000-01-01");
   # Enables Python Workers. Access to this flag is not restricted, instead bundles containing
   # Python modules are restricted in EWC.
   #
   # WARNING: Python Workers are still an experimental feature and thus subject to change.
 
+  pythonWorkersDevPyodide @58 :Bool
```

Comment:
I know it fits nicely under pythonWorkers but I still think it's better to have it be at the end of the file so that it's sorted by the flag id, next person to add a flag will be very confused as to why his receiving an error of 58 already in use.